matminer/featurizers/composition.py

```diff
@@ -2004,3 +2025,211 @@ def get_ideal_radius_ratio(self, n_neighbors):
         n = max(3, min(n_neighbors, 24))
 
         return self.ideal_ratio[n]
+
+
+class RoostFeaturizer(BaseFeaturizer):
+    """
+    Representation learning from stiochiometry
+    """
+
+    def __init__(self,
+        task="regression",
+        loss="L2",
+        model_name="roost",
+        elem_emb="matscholar",
+        elem_fea_len=64,
+        n_graph=3,
+        run_id=1,
+        seed=42,
+        epochs=100,
+        log=True,
+        optim="AdamW",
+        learning_rate=3e-4,
+        momentum=0.9,
+        weight_decay=1e-6,
+        batch_size=128,
+        workers=0, # load data in main process -- allows caching
+        device=torch.device("cuda") if torch.cuda.is_available() else torch.device("cpu"),
```

Comment:
This is one of the reasons we fail the tests. I am not sure how to work around this without #378